### ISSUE TITLE
Inherited permission issues with tablist.

### ISSUE THREAD

**Issue by johnsoncui02** [sgtcaze/NametagEdit#234]
# Bug Report

NameTagEdit Version: 4.2.0 (or known as 4.0.2 in spigot).....

Spigot Version: 1.12.2-R.0.1-SNAPSHOT

Plugins: Currently using luckperms with ezrankspro

Issue Explained: for example, rank A has rank B's permission, so on and so forth. Due to that reason. rank A has nte.A permission and rank B has nte.B permission. But the tablist does not seem to switch over but instead stays in rank A's tablist.

No errors in console

**Comment by JustOneMoreBlock** [comment]
Hello,

You either need to setup the order of the tags.

or

On `Rank B` negate the `Rank A` permission node.

Kind Regards,
Cory

**Comment by johnsoncui02** [comment]
![image](https://user-images.githubusercontent.com/23487437/35207000-5ddf8564-fef5-11e7-8ac2-16f50b5839cf.png)
pretty sure it's not the order of tags.

However, I'm unsure of how to negate permissions. Is there a way to automatically assume that say a person with priority 2 who also has a priority 1 permission be put on priority 1 name on tablist only?

**Comment by sgtcaze** [comment]
The order of your Groups.yml file is in the incorrect order.

Please [see this link](https://github.com/sgtcaze/NametagEdit/blob/master/documentation/Configuration.creole)

SortPriority and GroupOrder are different. SortPriority indicates how the nametags should be displayed in the tab. GroupOrder indicates the hiearchy of your groups (where Owner would be top priority, and default would be last)

Basically:
```
Owner:
   ...
Admin:
  ...

etc
```

If you continue experiencing issues, please reopen this issue (try tagging me, sometimes I do not see any updates on issues.)
